src/main/java/managers/DatabaseManager.java

```diff
@@ -0,0 +1,60 @@
+package managers;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+
+import objects.Database;
+
+public class DatabaseManager {
```

Comment:
This class seems to be doing the actual database work, but since it does not implement the IDatabase interface no other classes can access this via dependency inversion.